test/functional/apps/discover/group6/_view_mode_toggle.ts

```diff
@@ -125,7 +114,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
           await discover.selectTextBaseLang();
 
-          await testSubjects.existOrFail('dscViewModeToggle');
+          await testSubjects.missingOrFail('dscViewModeToggle');
```

Comment:
Would be great to also check that `discoverQueryTotalHits` is still present even without the tabs.

Reply:
Reverted here  [`dc99c68` (#197538)](https://github.com/elastic/kibana/pull/197538/commits/dc99c68dd97df4e6ccae2f559d012c89ed555103)